CDP4Common.NetCore.Tests/Helpers/NestedElementTreeGeneratorTestFixture.cs

```diff
@@ -108,12 +106,6 @@ public void SetUp()
                 Name = "Battery"
             };
 
-            this.elementDefinition_3 = new ElementDefinition(Guid.NewGuid(), this.cache, this.uri)
```

Comment:
why delete this?

Reply:
Because I added it yesterday, and Resharper told me it wasn't used ;).

Reply:
hahaha, ok, makes sense to me :-) apologies for being a but picky today